src/test/HostActivationTests/GivenThatICareAboutMultilevelSharedFxLookup.cs

```diff
@@ -366,6 +414,11 @@ public void Roll_Forward_On_No_Candidate_Fx_Fails_If_No_Higher_Version_Is_Availa
         [Fact]
         public void Roll_Forward_On_No_Candidate_Fx_Must_Look_Into_All_Lookup_Folders()
         {
+            if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Windows)
```

Comment:
Same as above - this should endup doing the right thing by design. Is that not the case per your changes?